test/unit/getProxySettings-test.js

```diff
@@ -0,0 +1,59 @@
+const { assert } = require('chai');
+
+const getProxySettings = require('../../lib/getProxySettings');
+
+
+describe('getProxySettings()', () => {
```

Comment:
I wonder what should happen when both `HTTP_PROXY` and `http_proxy` is set, if I have an enviornment where both are set (to different values), should one be discarded / presidence be given to a specific cased key?

We don't have a test case for this, perhaps worth adding even if both values should be present in the result.

Reply:
🤔 I think in this case logging should definitely reflect how `request` behaves in such case. Currently I have no idea how `request` resolves the precedence.

Reply:
Another GitHub issue? 😅

Reply:
https://github.com/apiaryio/dredd/issues/1310